es-core/src/ThemeData.h

```diff
@@ -83,7 +84,7 @@ class ThemeExtras : public GuiComponent
 private:
 	std::vector<GuiComponent*> mExtras;
 };
-
+*/
```

Comment:
It should be.  There is another similar commented out section that can be removed as well.